lib/util/emoji.spec.ts

```diff
@@ -21,6 +21,11 @@ describe('util/emoji', () => {
       expect(emojify(':foo: :bar: :bee:')).toBe(':foo: :bar: 🐝');
     });
 
+    it('convert warning shortcode to emoji', () => {
```

Comment:
Silly question incoming! 🫠 

Can we use the _good colorful emojis_ in the source files _directly_? So we can drop the _convert bad emoji to good emoji_ code? It feels a bit weird to have all this code dealing with emojis.

Reply:
No, because source file is located in the `node_modules`